include/freerdp/crypto/crypto.h

```diff
@@ -42,6 +42,10 @@ extern "C"
 	FREERDP_API void crypto_base64_decode(const char* enc_data, size_t length, BYTE** dec_data,
 	                                      size_t* res_length);
 
+	FREERDP_API char* crypto_base64url_encode(const BYTE* data, size_t length);
+	FREERDP_API void crypto_base64url_decode(const char* enc_data, size_t length, BYTE** dec_data,
```

Comment:
I´d keep the old return `char*` so you can check for failure (e.g. `NULL` return), just add the return size argument

Reply:
ok, sorry, did mix up the lines, looks like a case of `ENOTENOUGHCOFFEE`